### PR TITLE
ERM - Added derived table openaccess_publication_request

### LINKED ISSUE #784: ERM - Create derived table 'openaccess_publication_request' for Metadb
The purpose of this derived table is to show statistics associated with Open Access publication requests.
Attributes: https://wiki.folio.org/pages/viewpage.action?pageId=118271249

### PR DESCRIPTION
**Pull request by stdombek**
Closes #784 
This derived table shows data about publication requests from the FOLIO app Open Access. 

**Note for the reviewers:** The attribute `pr_group_fk` has no description, but has to part of the derived table.

### CODE REVIEW

File: sql_metadb/derived_tables/openaccess_publication_request.sql [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+COMMENT ON COLUMN openaccess_publication_request.pr_license IS 'UUID reference to a refdata value';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_license_value IS 'Refdata value for publication request license';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_license_label IS 'Refdata label for publication request license';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_doi IS 'DOI for the publication';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_group_fk IS '';
```
Comment (LauSchl) [inline sql_metadb/derived_tables/openaccess_publication_request.sql:147]: Is this going to be a placeholder for future changes?
Or is there another usecase for empty comments?

**Review by LauSchl — CHANGES_REQUESTED** [review]
Query itself works great.
The empty comments feels a bit redundant.

All queries:
- [x] PR Title and Description are accurate and thorough
- [x] PR is based on a new branch (not main)
- [x] PR scope is not overly broad
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented
- [] Table and column names are in all-lowercase
- [] Quotation marks are used only where necessary
- [x] JSON extraction is in standard form, for example:
      LDP:     t #>> '{f1,f2,f3}'    [for compatibility across LDP 1 & 2]
      Metadb:  jsonb_extract_path_text(t, f1, f2, f3)

Derived tables:
- [x] First line is "--metadb:table" directive, followed by blank line
- [x] User documentation in comment lines, followed by blank line
- [x] File name is listed in `runlist.txt` after dependencies

File: sql_metadb/derived_tables/openaccess_publication_request.sql (hunk 1 above)
Comment (stdombek) [reply sql_metadb/derived_tables/openaccess_publication_request.sql:147]: It is a placeholder. We have to clarify this in one of our next working group meetings. The change can then be made very quickly afterwards.

**Review by VandanaShah-Cornell — APPROVED** [review]
All queries:
- [x] PR Title and Description are accurate and thorough
- [x] PR is based on a new branch (not main)
- [x] PR scope is not overly broad
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary
- [x] JSON extraction is in standard form, for example:
      LDP:     t #>> '{f1,f2,f3}'    [for compatibility across LDP 1 & 2]
      Metadb:  jsonb_extract_path_text(t, f1, f2, f3)


Derived tables:
- [x] First line is "--metadb:table" directive, followed by blank line
- [x] User documentation in comment lines, followed by blank line
- [x] File name is listed in `runlist.txt` after dependencies

**Review by LauSchl — APPROVED** [review]

**Pull request merged**